tests/pipeline/test_blaze.py

```diff
@@ -1365,49 +1414,73 @@ def test_deltas_only_one_delta_in_universe(self, asset_info):
             'value': [10, 11],
         })
         deltas = bz.data(deltas, name='deltas', dshape=self.dshape)
-        expected_views = keymap(pd.Timestamp, {
+        expected_views_all_deltas = keymap(pd.Timestamp, {
             '2014-01-02': np.array([[0.0, 11.0, 2.0],
                                     [1.0, 2.0, 3.0]]),
             '2014-01-03': np.array([[10.0, 2.0, 3.0],
                                     [2.0, 3.0, 4.0]]),
             '2014-01-04': np.array([[2.0, 3.0, 4.0],
                                     [2.0, 3.0, 4.0]]),
         })
+        expected_views_novel_deltas = keymap(pd.Timestamp, {
```

Comment:
can you add a comment calling out the `(row_ix, col_ix)` of the difference so people don't miss it.